### PR TITLE
doc(dm): add dm v5.3.0 changelog

### PR DESCRIPTION
**Pull request by Ehco1996**
### What is changed and how it works?
as title

### Check List 

Tests 

 - No code

Code changes

- None

Side effects

- None

Related changes

- None

### Release note 

```release-note
Please add a release note.
`None`.
```

### CODE REVIEW

**Review by lichunzhu — APPROVED** [review]

**Comment by Ehco1996** [comment]
could you please take another look for this pr ? thanks 🧡🧡🧡 @GMHDBJD

**Review by GMHDBJD — APPROVED** [review]
LGTM

File: dm/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,48 @@
 
 All notable changes to this project will be documented in this file.
 
+## [5.3.0] 2021-11-30
```
Comment (GMHDBJD) [inline dm/CHANGELOG.md:5]: Wrond date?

**Comment by GMHDBJD** [comment]
/hold
I think we should merge after v5.3.0 release

**Comment by Ehco1996** [comment]
> /hold I think we should merge after v5.3.0 release

Sounds reasonable

**Comment by Ehco1996** [comment]
/unhold

v5.3.0 has released

File: dm/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,48 @@
...
+- Load and sync units support more monitoring metrics [#1778](https://github.com/pingcap/dm/pull/1778)
+- Support for concurrently manipulating tasks via dmctl [#1995](https://github.com/pingcap/dm/pull/1955)
+- Optimize DML concurrency for incremental replication [#2043](https://github.com/pingcap/dm/pull/2043)
+- Prompt user when HTTP proxy-related environment variables are detected [#1960](https://github.com/pingcap/dm/pull/1960)
+- Optimize log display when handling RowEvent [#2006](https://github.com/pingcap/dm/pull/2006)
+- Optimize log display when SQL execution is too slow [#2024](https://github.com/pingcap/dm/pull/2024)
+- Optimize logic for fetching status data from upstream sources to reduce pressure on upstream [#2076](https://github.com/pingcap/dm/pull/2076)
+- Report errors and prompt users when encountering unsupported binlog formats [#2099](https://github.com/pingcap/dm/pull/2099)
+- Support for batch manipulation of all sync tasks in a data source via dmctl [#2166](https://github.com/pingcap/dm/pull/2166)
+- Generate DML WHERE statements from downstream table schemas [#3168](https://github.com/pingcap/ticdc/pull/3168)
```
Comment (glorv) [inline dm/CHANGELOG.md:40]: Please remote this line, this feature si not available in v5.3.0

Comment (glorv) [reply dm/CHANGELOG.md:40]:
```suggestion
```

**Pull request merged**